integration/client_sdk/go/utils/utils.go

```diff
@@ -110,7 +110,7 @@ func populateWallet(wallet *gateway.Wallet) error {
 	return wallet.Put("appUser", identity)
 }
 
-func setupNetwork(channel string) (*gateway.Network, error) {
+func SetupNetwork(channel string) (*gateway.Network, error) {
```

Comment:
As these functions seem to be generic and useful, would it make sense to move them (and perhaps the entire utils.go) out of `integration`?

Reply:
Yes, I agree at some point I think it would be convenient. I told @mbrandenburger the same thing. Maybe as part of the discussions for the merge and work after? If you accept this merge, that change wouldn't affect that much what I am doing. There would only be 1 line of code that would have to be modified, the import.